src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -48,6 +51,8 @@
     private final BigQueryPdao bigQueryPdao;
     private final RestHighLevelClient client;
 
+    private final Map<String, String> timMap = Map.of("example_now", "example:identifier.now");
```

Comment:
Suggestions for how to achieve this?

Reply:
I forgot to mention in standup, I have a change (and now a PR) for a fix that will only modify names if they need encoding. https://github.com/DataBiosphere/jade-data-repo/pull/956

Reply:
I only do a replacement if the there's a corresponding value in the `timMap` (otherwise it will return `null`).

Reply:
Using the new encode() method will remove the need to look up names in a map. If name needs to be encoded, it will be, otherwise nothing happens. So you'll be able to call `encode()` on all columns.

Reply:
Perhaps instead of operating on the SQL query I should operate on the object returned from BigQuery, that way I can easily loop over column names and avoid using a regex.

Reply:
If we want to support users passing queries to the API that have TIM names in them, I think we have to modify the query before passing it to BQ. Another approach would be to add another API parameter that provides a column name mapping for all columns in the result, and only use TIM names there. We may in the long term support a richer API that hides the SQL query from users, and at that point we'd have the information to avoid parsing the SQL contents. I think a heuristic based approach where we parse the SQL using a regex is fine for now.